supertokens_python/querier.py

```diff
@@ -166,11 +166,18 @@ async def f(url: str) -> Response:
 
         return await self.__send_request_helper(path, "POST", f, len(self.__hosts))
 
-    async def send_delete_request(self, path: NormalisedURLPath):
+    async def send_delete_request(
```

Comment:
If an empty API key is passed during dashboard init then the sign up flow should use email password